vm/systemSmartContracts/validator_test.go

```diff
@@ -466,6 +466,66 @@ func TestStakingValidatorSC_ExecuteStakeTooManyNodes(t *testing.T) {
 		assert.Equal(t, entry.Topics[0], []byte(numberOfNodesTooHigh))
 	}
 
+	stakeCalledInStakingSC := false
+	eei.ExecuteOnDestContextCalled = func(destination, sender []byte, value *big.Int, input []byte) (*vmcommon.VMOutput, error) {
+		if strings.Contains(string(input), "stake") {
+			stakeCalledInStakingSC = true
+			assert.False(t, stakeCalledInStakingSC)
+		}
+		return &vmcommon.VMOutput{}, nil
+	}
+
+	key1 := []byte("Key1")
+	key2 := []byte("Key2")
+	key3 := []byte("Key3")
+	arguments.Function = "stake"
+	arguments.CallValue = big.NewInt(0).Mul(big.NewInt(3), big.NewInt(10000000))
+	arguments.Arguments = [][]byte{big.NewInt(3).Bytes(), key1, []byte("msg1"), key2, []byte("msg2"), key3, []byte("msg3")}
```

Comment:
Test is called TestStakingValidatorSC_ExecuteStakeTooManyNodes, but here we don't check that the number of nodes is too high. Can we change the limit nodes to 3 in this case and check that this still works?

Reply:
It checks. as AddLogEntry must be called. Line 465 and 485

Reply:
Right, missed it